hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/records/DoubleWayMultiPathsRecords.java

```diff
@@ -38,13 +38,12 @@
 
 public abstract class DoubleWayMultiPathsRecords extends AbstractRecords {
 
-    protected final Stack<Record> sourceRecords;
-    protected final Stack<Record> targetRecords;
+    private final Stack<Record> sourceRecords;
+    private final Stack<Record> targetRecords;
 
-    protected Record currentRecord;
     private IntIterator lastRecordKeys;
-    protected int current;
-    protected boolean forward;
+    private int current;
```

Comment:
means what

Reply:
mean int code of currently accessed vertex's Id